frontend/components.d.ts

```diff
@@ -74,6 +74,9 @@ declare module 'vue' {
     IssueSubscriberPanel: typeof import('./src/components/IssueSubscriberPanel.vue')['default']
     IssueTable: typeof import('./src/components/IssueTable.vue')['default']
     IssueTaskStatementPanel: typeof import('./src/components/IssueTaskStatementPanel.vue')['default']
+    KBarFooter: typeof import('./src/components/KBar/KBarFooter.vue')['default']
```

Comment:
Let's remove this from this PR (Yes, it does appear in my repo as well, but would be better to put in a separate PR)

Reply:
Np. Just revert these frontend files.